symphony-bdk-examples/bdk-app-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -3,18 +3,27 @@ bdk:
   host: acme.symphony.com
   bot:
     username: mybot
-    privateKeyPath: /Users/user.name/.symphony/privatekey.pem
+    privateKeyPath: ${user.home}/.symphony/keys/rsa-private-bot.pem
+
   app:
     appId: myapp
-    privateKeyPath: /Users/user.name/.symphony/privatekey.pem
+    privateKeyPath: ${user.home}/.symphony/keys/app_privatekey.pem
 
 bdk-app:
-  auth:
+  auth.enabled: true
+  auth.jwtCookie:
```

Comment:
Again, yaml allows hierarchical properties: 
```yaml
bdk-app:
  auth:
    enabled: true
    jwtCookie: ...
```
Otherwise you should use `application.properties` to have: 
```
bdk-app.auth.enabled=true
bdk-app.auth.jwtCookie.enabled=true
```